core/src/main/java/org/apache/gravitino/listener/TagEventDispatcher.java

```diff
@@ -70,8 +82,10 @@ public String[] listTags(String metalake) {
   public Tag[] listTagsInfo(String metalake) {
     // TODO: listTagsInfoPreEvent
     try {
-      // TODO: listTagsInfoEvent
-      return dispatcher.listTagsInfo(metalake);
+      Tag[] tags = dispatcher.listTagsInfo(metalake);
+      eventBus.dispatchEvent(
+          new ListTagInfoEvent(PrincipalUtils.getCurrentUserName(), metalake, tags));
+      return tags;
     } catch (Exception e) {
       eventBus.dispatchEvent(
           new ListTagsInfoFailureEvent(PrincipalUtils.getCurrentUserName(), metalake, e));
```

Comment:
could you rename `ListTagsInfoFailureEvent` to `ListTagInfoFailureEvent` to keep consistent with other list event?